CHANGES.md

```diff
@@ -1,4 +1,5 @@
 ## Unreleased
+- Address warning 32 errors in ci (@bikallem #876)
```

Comment:
We don't to describe CI changes in the change log.

Reply:
removed references to ci